src/containers/blocks.jsx

```diff
@@ -461,8 +472,8 @@ const mapStateToProps = state => ({
         state.scratchGui.mode.isFullScreen
     ),
     extensionLibraryVisible: state.scratchGui.modals.extensionLibrary,
-    locale: state.intl.locale,
-    messages: state.intl.messages,
+    locale: state.scratchGui.locales.locale,
+    messages: state.scratchGui.locales.messages[state.scratchGui.locales.locale],
```

Comment:
Can't the reducer do this work for everyone, so we can continue to refer to just `state.scratchGui.messages`?